README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
Overall that sounds good to me. One more thing though, what happens if the connector has a single more specific route with that same prefix? One account would mess up that caching mechanism because it would be technically incorrect to say that _all_ accounts under that prefix share the same liquidity curve, even if you know that 10,000 do and 1 doesn't.